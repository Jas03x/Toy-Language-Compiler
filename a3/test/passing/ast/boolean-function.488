% Declaring and calling a function that returns a boolean

{
    function test : integer
    {
        return with 1
    }

    write test
}
