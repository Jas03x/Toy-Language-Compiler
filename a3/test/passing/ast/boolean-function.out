 { 
 function test : integer { 
 return with 1
 } 
 write test
 }
